stories/General/ButtonStory.tsx

```diff
@@ -44,6 +44,21 @@ const buttonProps = [
     require: 'no',
     description: 'Sets Button to small version.',
   },
+  {
```

Comment:
Could we also show an example component that uses `tag="a"` since that will likely be a common use case.

Reply:
Added!